test/decl/func/functions.swift

```diff
@@ -179,3 +179,29 @@ func parentheticalInout2(_ fn: (((inout Int)), Int) -> ()) {
   var value = 0
   fn(&value, 0)
 }
+
+class A {}
```

Comment:
I think test/Constraints/members is a better place for these tests

Reply:
Thanks, I have moved it to that file now 👍